code/game/gamemodes/clock_cult/clock_items/clockwork_slab.dm

```diff
@@ -327,57 +329,238 @@
+			as the amount of servants in the covenant increase; additionally, slabs can only produce components when held by a servant, and holding more than one slab will cause both \
+			of them to halt progress until one of them is removed from their person.<br><br>"
+			dat += "Your slab has an internal storage of components, but it isn't meant to be the main one. Instead, there's a <b>global storage</b> of components that can be \
+			added to through various ways. Anything that needs components will first draw them from the global storage before attempting to draw them from the slab. Most methods of \
+			component production add to the global storage. You can also offload components from your slab into the global storage by using it on a tinkerer's cache, a structure whose \
+			primary purpose is to do just that (although it will also slowly produce components when placed near a brass wall.)<br><br>"
+			dat += "<font color=#BE8700 size=3>-=-=-=-=-=-</font>"
+		if("Scripture")
+			dat += "<font color=#BE8700 size=3>The Ancient Scripture</font><br><br>"
+			dat += "If you have experience with the Nar-Sian cult (or the \"blood cult\") then you will know of runes. They are the manifestations of the Geometer's power, and where most \
+			of the cult's supernatural ability comes from. The servant equivalent of runes is called <b>scripture</b>, and unlike runes, scripture is loaded into your clockwork slab.<br><br>"
+			dat += "Each piece of scripture has widely-varying effects. Your most important scripture, <i>Geis</i>, is obvious and suspicious, but charges your slab with energy and allows \
+			you to attack a non-servant in melee range to restrain them and begin converting them into a servant. This is just one example; each piece of scripture can be simple or \
+			complex, be obvious or have hidden mechanics that can only be found through trial and error.<br><br>"
+			dat += "Any given piece of scripture has a component cost listed in its \"Recite\" button. The acronyms for the components should be obvious if you've read about components \
+			already; reciting this piece of scripture will consume the listed components, first from the global storage and then from your slab. Note that failing to recite a piece of \
+			scripture will <i>not</i> consume the components required to recite it.<br><br>"
+			dat += "It should also be noted that some scripture cannot be recited alone. Especially with more powerful scripture, you may need multiple servants to recite a piece of \
+			scripture; both of you will need to stand still until the recital completes. <i>Only human and silicon servants are valid for scripture recital!</i> Constructs cannot help \
+			in reciting scripture.<br><br>"
+			dat += "Finally, scripture is separated into four \"tiers\" based on power: Drivers, Scripts, Applications, and Judgement.[prob(1) ? "(If you're looking for Revenant, don't \
```

Comment:
this isn't as bad as replacing geis randomly but I still think it might be slightly confusing in the unlikely scenario that a new player has it appear